### PR TITLE
Pathway show

### PR DESCRIPTION
**Pull request by SafiyahZ**
Follows the design on figma but feedback would be appreciated

### CODE REVIEW

**Comment by CraigNorford** [comment]
Make sure merge main from into this branch to avoid more node_modules related modules

File: app/assets/stylesheets/pathways/_show_page.scss [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+.Difficulty.intermediate {
+   background-color: rgb(252, 209, 156);
+}
+
+.Difficulty.advanced {
+   background-color: rgb(255, 172, 167);
+}
+
+.title4 {
```
Comment (CraigDoesCode) [inline app/assets/stylesheets/pathways/_show_page.scss:36]: is title4 a descriptive class name?

File: app/assets/stylesheets/pathways/_show_page.scss [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+   font-size: 30px;
+   font-weight: bold;
+}
+
+.c {
```
Comment (CraigDoesCode) [inline app/assets/stylesheets/pathways/_show_page.scss:41]: is .c a descriptive class name?

File: app/assets/stylesheets/pathways/_show_page.scss [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+.Title {
+   font-size: 50px;
+   font-weight: bold;
+   letter-spacing: 2px;
+   margin-bottom: 10px;
+}
+
+.Difficulty {
```
Comment (CraigDoesCode) [inline app/assets/stylesheets/pathways/_show_page.scss:13]: class names should be lowercase and hyphenated ".this-class" or ".class"

File: app/assets/stylesheets/pathways/_show_page.scss [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+   background-color: rgb(51, 85, 209);
+   height: 50px;
+   width: 200px;
+   border-radius: 20px;
+   text-align: center;
+   padding-top: 14px;
+   font-size: smaller;
+   margin-left: 15px;
+}
+
```
Comment (CraigDoesCode) [inline app/assets/stylesheets/pathways/_show_page.scss:84]: .create and .add are identical apart from the margin left how can we keep this DRY?

**Comment by CraigDoesCode** [comment]
1. improve naming conventions heres a great article https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/
2. consider that you are using scss and are able to store variables which will make your easier to write i.e storing colours  https://sass-lang.com/documentation/variables

**Review by CraigDoesCode — APPROVED** [review]

**Pull request merged**